File: facesdk_cs_tests/facesdk_cs_tests/PathsConfig.cs
using System.Net;

namespace facesdk_cs_tests
{
    public static class PathsConfig
    { static PathsConfig()
        {
            ServicePointManager.ServerCertificateValidationCallback += 
                (sender, cert, chain, sslPolicyErrors) => true;
        }
        public static readonly FaceSdk FaceSdk = new FaceSdk(Environment.GetEnvironmentVariable("ServiceLink") ?? "http://localhost:41101/");
        private static readonly string ProjectDir = GetProjectDirectory();
        private static readonly string FilesPath = Path.Combine(ProjectDir, "misc", "files");
        public static readonly string Face1Path = Path.Combine(FilesPath, "face1.jpg");
        public static readonly string Face2Path = Path.Combine(FilesPath, "face2.jpg");
        public static readonly string Face3Path = Path.Combine(FilesPath, "face3.jpg");
        public static readonly string LivePhotoPath = Path.Combine(FilesPath, "me.png");
        public static readonly string PrintedDocumentPath = Path.Combine(FilesPath, "printedDoc.png");
        public static readonly string DocumentWithLivePath = Path.Combine(FilesPath, "me_and_id.png");
        public static readonly string SeveralFacesImagePath = Path.Combine(FilesPath, "severalFaces.jpg");


        private static string GetProjectDirectory()
        {
            var assemblyLocation = Assembly.GetExecutingAssembly().Location;
            var binDirectory = Path.GetDirectoryName(assemblyLocation);
            var projectDirectory =
                Path.GetFullPath(Path.Combine(binDirectory ?? throw new InvalidOperationException(),
                    "../../../"));

            return projectDirectory;
        }

        public static byte[] ReadImageBytes(string path)
        {
            if (!File.Exists(path))
            {
                throw new FileNotFoundException($"File not found at path: {path}");
            }
            return File.ReadAllBytes(path);
        }

    }
}

File: facesdk_java_tests/src/test/java/PathsConfig.java
import com.regula.facesdk.webclient.FaceSdk;

import java.io.Console;
import java.io.IOException;
import java.nio.file.Files;
import java.nio.file.Path;
import java.nio.file.Paths;
import java.lang.System;

public class PathsConfig {
    public static final String API_BASE_PATH = System.getProperty("apiBasePath", "http://localhost:41101/");
    static final FaceSdk faceSdk = new FaceSdk(PathsConfig.API_BASE_PATH);
    private static final String PROJECT_DIR = System.getProperty("user.dir");
    private static final Path FILES_PATH = Paths.get(PROJECT_DIR, "misc", "files");
    public static final Path FACE1_PATH = FILES_PATH.resolve("face1.jpg");
    public static final Path FACE2_PATH = FILES_PATH.resolve("face2.jpg");
    public static final Path FACE3_PATH = FILES_PATH.resolve("face3.jpg");
    public static final Path LIVE_PHOTO_PATH = FILES_PATH.resolve("me.png");
    public static final Path PRINTED_DOCUMENT_PATH = FILES_PATH.resolve("printedDoc.png");
    public static final Path DOCUMENT_WITH_LIVE_PATH = FILES_PATH.resolve("me_and_id.png");
    public static final Path SEVERAL_FACES_IMAGE_PATH = FILES_PATH.resolve("severalFaces.jpg");

    public static byte[] readImageBytes(Path path) throws IOException {
        return Files.readAllBytes(path);
    }
}
